[2024-09-11T10:22:39.224+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-11T10:22:39.371+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: max_min_dag.generate_random_numbers scheduled__2024-09-10T00:00:00+00:00 [queued]>
[2024-09-11T10:22:39.404+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: max_min_dag.generate_random_numbers scheduled__2024-09-10T00:00:00+00:00 [queued]>
[2024-09-11T10:22:39.406+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 4
[2024-09-11T10:22:39.475+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_random_numbers> on 2024-09-10 00:00:00+00:00
[2024-09-11T10:22:39.518+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=69) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-11T10:22:39.527+0000] {standard_task_runner.py:72} INFO - Started process 70 to run task
[2024-09-11T10:22:39.524+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'max_min_dag', 'generate_random_numbers', 'scheduled__2024-09-10T00:00:00+00:00', '--job-id', '154', '--raw', '--subdir', 'DAGS_FOLDER/simple_dag.py', '--cfg-path', '/tmp/tmpw4ttx7zq']
[2024-09-11T10:22:39.529+0000] {standard_task_runner.py:105} INFO - Job 154: Subtask generate_random_numbers
[2024-09-11T10:22:49.643+0000] {cli_action_loggers.py:177} WARNING - Failed to log action (psycopg2.OperationalError) could not translate host name "postgres" to address: Temporary failure in name resolution

(Background on this error at: https://sqlalche.me/e/14/e3q8)
